docs/releases/5.1.txt

```diff
@@ -112,7 +112,7 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   720,000 to 870,000.
 
-* The default ``parallelism`` of the ``ScryptPasswordHasher`` is 
+* The default ``parallelism`` of the ``ScryptPasswordHasher`` is
```

Comment:
Please revert this change

Reply:
Sorry about that. PyCharm auto-formatted the file and I didn't notice. I'll revert it.